test/prism_regression/begin.rb

```diff
@@ -0,0 +1,6 @@
+# typed: true
+
+begin
```

Comment:
Could you also add a test case to ensure that empty bodies (to exercise the `node->body == nullptr` case) parse correctly, too?

```suggestion
begin
end

begin
```